Documentation/email-configuration.md

```diff
@@ -20,6 +20,7 @@ set `auth` to `plain` and specify your username and password.
     "host": "smtp.example.org",
     "port": 587,
     "auth": "plain",
```

Comment:
Changed this to use auth if "username" and "password" fields are provided.

Also going to let the port be provided in the "host" field.